scripts/CEE/check-tech-preview-features/script.sh

```diff
@@ -23,7 +23,7 @@ getFeatureGates() {
 }
 
 getAdditionalFeatureGates() {
-arg=(${1})
+arg=("${1}")
```

Comment:
I haven't checked but not sure if this change will break the function. Please have a test.

If something written in the past is hard to fix, we might ignore the shellcheck error for now to unblock the PR.
https://www.shellcheck.net/wiki/Ignore